Tests/Framework/Storage/StorageTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Microsoft.Xna.Framework.Storage;
+using Microsoft.Xna.Framework;
+using System.ComponentModel.Design;
+
+namespace MonoGame.Tests.Framework
+{
+    [TestFixture]
+    public class StorageDeviceTests
+    {
+        const string MY_GAME = "MyGame";
+
+        [Test]
+        public void IsConnected_ShouldReturnFalse_OnException()
+        {
+            var device = new StorageDevice(null);
+
+            // We haven't opened a container yet, so should be false
+            Assert.IsFalse(device.IsConnected);
```

Comment:
@ThomasFOG Do you find `IsConnected` to be useful in a modern API for storage?  Is there any case now where save storage is disconnected?

Reply:
No, I don't think so. All systems ensure that storage device is always available nowadays.

Reply:
I'll remove it.